LabSources/SampleScripts/Introduction/07.3 Exchange 2019 Server and client, domain joined.ps1

```diff
@@ -0,0 +1,28 @@
+# The is similar to the '05 SQL Server and client, domain joined.ps1' but installs an Exchange 2019 server instead
+# of a SQL Server.
+# IMPORTANT NOTE: You must have Exchange 2019 ISO or CU already available. Microsoft has limited Exchange 2019 access to VL & MSDN
+# so it is not a publicly available for download. Also note Exchange 2019 will only run on Windows 2019 Core or Desktop but not Nano
+
+$labName = 'LabEx2019'
+
+#create an empty lab template and define where the lab XML files and the VMs will be stored
+New-LabDefinition -Name $labName -DefaultVirtualizationEngine HyperV -VmPath I:\AutomatedLab-VMs
```

Comment:
Can you remove the 'VmPath' parameter? This should not be set in the sample scripts. Thanks!